src/include/kernel/abi.h

```diff
@@ -30,7 +30,7 @@
 /** \brief SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
 #define SOF_ABI_MINOR 28
-#define SOF_ABI_PATCH 0
+#define SOF_ABI_PATCH 1
```

Comment:
Whay are we touching IPC3, we've not changed any IPC3 here ?

Reply:
To make CI / checkpatch happy

Reply:
Not following either, these tools shouldn't care.